### ISSUE TITLE
nullptr_t comparsion

### ISSUE THREAD

**Issue by mikcza** [ar90n/msgpack11#2]
When building v0.0.7 for Android with NDK 15.0.4075724 using Clang I got a following error message:

```
msgpack11.cpp:360:58: error: invalid operands to binary expression ('nullptr_t' and 'nullptr_t')
    return is_less_type || (is_same_type && (m_value < static_cast<const Value<tag, T> *>(other)->m_value)); 

msgpack11.cpp:623:5: note: in instantiation of member function 'msgpack11::Value<msgpack11::MsgPack::Type::NUL, nullptr_t>::less' requested here
    MsgPackNull() : Value(nullptr) {}  
```

**Comment by ar90n** [comment]
Hi mikcza.
Thanks for your report.
I fixed this issue at v0.0.8.
It seems that it is same to https://github.com/dropbox/json11/issues/91

Please checkout and try it !

**Comment by mikcza** [comment]
It works now. Thank you.
